src/js/components/CustomCardControl/CustomCardControl.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { CardControl } from 'cf-component-card';
+import { Button } from 'cf-component-button';
+import { CLOUDFLARE_UPGRADE_PAGE } from '../../constants/UrlPaths.js';
+
+class CustomCardControl extends Component {
+    static propTypes = {
+        name: PropTypes.string,
+    };
+
+    render() {
+        let { integrationName } = this.props;
+
+        // TODO: Add minimumPlan to lang folder 
+        let planList = {
+            "Free Website": 0,
+            "Pro Plan": 1,
+            "Biz Plan": 2,
+            "Ent Plan": 3
+        };
```

Comment:
GET /zones returns a plan object:
```
"plan": { "id": "e592fd9519420ba7405e1307bff33214", "name": "Pro Plan", "price": 20, "currency": "USD", "frequency": "monthly", "legacy_id": "pro", "is_subscribed": true, "can_subscribe": true }
```

Lets create a constants/plans.js with an object that has this structure:
```
{
    id: "e592fd9519420ba7405e1307bff33214",
   legacy_id: "pro"
   name: "proPlan"
   rank: 1
}
```
The name of the plan needs to be localized so in lang/en.js do make keys along the lines of `plans.proPlan` so that its easy to get the localized value from that plan JSON later.